main.js

```diff
@@ -294,6 +324,36 @@ export class IdDecoder {
         bitLength: this.fixedBitLength
       });
     }
+    if(this.multihash) {
+      // <varint hash fn code>: identity function
+      const [hashFnCode] = decoded;
+
+      if(hashFnCode !== MULTIHASH_IDENTITY_FUNCTION_CODE) {
+        throw new Error('Invalid multihash function code.');
+      }
+      // <varint digest size in bytes>
+      const digestSize = decoded[1];
+
+      if(digestSize > 127) {
+        throw new Error(`Decoded identifier size too large.`);
```

Comment:
Fix quotes and use RangeError.
```suggestion
        throw new RangeError('Decoded identifier size too large.');
```

Reply:
Updated fcd0ad9